pkg/cmd/server/kubernetes/master_config.go

```diff
@@ -109,6 +110,13 @@ func BuildKubernetesMasterConfig(options configapi.MasterConfig, requestContextM
 
 	admissionController := admission.NewFromPlugins(kubeClient, strings.Split(server.AdmissionControl, ","), server.AdmissionControlConfigFile)
 
+	// we need to set some custom parameters on the service account admission controller, so create that one by hand
+	saAdmitter := saadmit.NewServiceAccount(kubeClient)
+	if options.ServiceAccountConfig.AllowPermissive {
+		saAdmitter.AllowAllSecretsAnnotation = configapi.ServiceAccountAllowAnySecretAnnotation
+	}
+	admissionController = admission.NewChainHandler(admissionController, saAdmitter)
```

Comment:
> SCC depends on it being set, and quota should come last since a later rejection will allocate quota until the reconcile controller frees it

You know what we need: a dependency graph.... :)